client/src/helpers/nova/novaTimeUtils.ts

```diff
@@ -0,0 +1,58 @@
+import { ProtocolInfo } from "@iota/sdk-wasm-nova/web";
+
+export function unixTimestampToSlotIndexConverter(protocolInfo: ProtocolInfo): (unixTimestampSeconds: number) => number {
```

Comment:
would there be a way to having to create a curried function with `protocolInfo` everything one of these functions wants to be used?

Reply:
I'll add a proposed solution with a custom hook